twitchpopups.htm

```diff
@@ -1,6 +1,35 @@
 <!DOCTYPE HTML>
 <html>
 <head>
+
+    <script type="text/javascript">
+
+        // Change Limmy to your Twitch channel
+        var twitchChannel = 'Limmy';
+
+        // Your alert background. Default is a vibrant green
+        var alertBg = '#00AA00';
+
+        // Hotseat background colour. Default is a firey, firey orange.
+        var hotseatBg = '#aa1100';
+
+        // The emoji that surrounds the hotseat messages.
+        var hotseatEmoji = '🔥';
+        // To open an emoji keyboard on your computer:
+        //     Windows: Windows Key + Period (.)

```

Comment:
@DaftLimmy most programmers (or streamers) are not native English speakers and will be more familiar with American English